mobilecoind/api/proto/mobilecoind_api.proto

```diff
@@ -8,6 +8,8 @@ syntax = "proto3";
 import "google/protobuf/empty.proto";
 import "external.proto";
 import "blockchain.proto";
+import "watcher.proto";
+import "ledger.proto";
```

Comment:
This proto file now needs to depend on the fog ledger proto file, which is not the most elegant but I don't think it causes any issues (and saves us from having to duplicate the `TxOutResult` message it needs from there).